lib/pure/math.nim

```diff
@@ -880,6 +880,32 @@ func floorMod*[T: SomeNumber](x, y: T): T =
   result = x mod y
   if (result > 0 and y < 0) or (result < 0 and y > 0): result += y
 
+func euclDiv*[T: SomeInteger](x, y: T): T {.since: (1, 5, 1).} =
```

Comment:
if this is not {.inline.}, inlining cannot happen unless:
* calling code is in same module
* passc:-flto is passed, which can be 3X slower to compile and can also result in slower binaries

if this is {.inline.}, C compiler is free to ignore the hint

see https://github.com/nim-lang/RFCs/issues/198#issuecomment-596968252 for explanations and benchmarks

Reply:
pre-existing floorDiv and lots of friends are not inline. It could be cleaned in the following PR.

Reply:
ok